rotkehlchen/db/filtering.py

```diff
@@ -195,12 +196,15 @@ class DBTransactionsPendingDecodingFilter(DBFilter):
     """
     This filter is used to find the ethereum transactions that have not been decoded yet
     using the query in `TRANSACTIONS_MISSING_DECODING_QUERY`. It allows filtering by chain.
+
+    Due to the joining we need to check for either value being NULL (meaning no entry
+    in evm_tx_mappings) or not having the DECODED value.
     """
     chain_id: ChainID | None
 
     def prepare(self) -> tuple[list[str], list[Any]]:
-        query_filters = ['B.tx_id is NULL']
-        bindings: list[int] = []
+        query_filters = ['(B.tx_id IS NULL OR B.tx_id NOT IN (SELECT tx_id FROM evm_tx_mappings WHERE value=?))']  # noqa: E501
+        bindings: list[int] = [HISTORY_MAPPING_STATE_DECODED]
```

Comment:
Can we also move this `HISTORY_MAPPING_STATE_DECODED` and rename it? Is not related to history events but transactions and is part of what lead to the confusion yesterday

Reply:
From the PR comments: 

>Also need to (in another commit) figure out the usages of those flags and at least rename them to avoid confusion.